lambdas/sqs-message-remover/src/index.js

```diff
@@ -4,17 +4,40 @@ const get = require('lodash/get');
 
 const { sqs } = require('@cumulus/aws-client/services');
 const { deleteSQSMessage } = require('@cumulus/aws-client/SQS');
+const { deleteS3Object } = require('@cumulus/aws-client/S3');
 const {
   getSfEventMessageObject,
   getSfEventStatus,
   isFailedSfStatus,
   isSfExecutionEvent,
   isTerminalSfStatus,
 } = require('@cumulus/common/cloudwatch-event');
+const { envUtils } = require('@cumulus/common');
 const Logger = require('@cumulus/logger');
 
 const logger = new Logger({ sender: '@cumulus/sqs-message-remover' });
 
+/**
+ * Deletes archived SQS Message from S3
+ *
+ * @param {Object} messageId - SQS message ID
+ * @returns {void}
+ */
+async function deleteArchivedMessage(messageId) {
```

Comment:
I think it might be good to move this function and `archiveMessage` to the `ingest` package (`ingest/sqs.ts`?) and add unit tests for them explicitly. The other reason I like having those two functions in the same file is that we can centralize this logic for getting the message key into a third function:

`${stackName}/archived-incoming-messages/${messageId}`

Otherwise, it just seems too easy for bugs to happen with a typo in the S3 key in either function.

Would also rename `archiveMessage` to `archiveSqsMessageToS3` and `deleteArchivedMessage` to `deleteArchivedMessageFromS3`

Reply:
[`5223038` (#2132)](https://github.com/nasa/cumulus/pull/2132/commits/522303829adf7b5c63fec589dced18ebcf084cf4)